cmd/vicadmin/vicadm.go

```diff
@@ -483,6 +483,9 @@ func vSphereSessionGet(sessconfig *session.Config) (*session.Session, error) {
 		log.Errorf("Got %s while creating user session", err)
 		return nil, err
 	}
+	if usersession == nil {
+		return nil, fmt.Errorf("UserSession from vSphere came back nil but no error was reported")
```

Comment:
@gigawhitlocks the only case where we're aiming for a degraded dashboard is if vic-admin cannot connect to vSphere to validate supplied credentials and the only thing we need to report at that time is that we cannot connect to vsphere.

The only reason to do this via dashboard was to provide some additional diagnostics as to _why_ we cannot connect but if it's causing issues then a message similar to that for internal server error suffices to let the user know why they cannot authenticate.